api/models/period.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
```

Comment:
Yeah lol I thought about using Mongoose, but found that the amount of code I'd have to write was ~relatively~ similar and it decreased flexibility bc of its rigid schema. For example, since we have log in with Google AND log in with password, some users have a hash field while others have a google_id field. Mongoose would have forced me to have all the fields. I don't mind using it though I prefer using the native driver, but I see Mongoose might obviously simplify some things.